modules/nw-ovn-k-anp-troubeshooting.adoc

```diff
@@ -0,0 +1,36 @@
+//module included in the following assemblies:
+//
+//networking/network_security/AdminNetworkPolicy/nw-ovn-k-anp-troubleshooting.adoc
+:_mod-docs-content-type: REFRENCE
+[id="anp-troubleshooting_{context}"]
+= Checking creation of ANP
+
+To check that your `AdminNetworkPolicy` (ANP) and `BaselineAdminNetworkPolicy` (BANP) are created correctly, check the status outputs of the following commands: `oc describe anp` or `oc describe banp`.
+
+A good status will indicate that `OVN DB plumbing was successful` and that `SetupSucceeded`
+
+.Example ANP with a good status
+[source,terminal]
+----
+...
+Conditions:
+Last Transition Time:  2024-06-08T20:29:00Z
+Message:               Setting up OVN DB plumbing was successful
+Reason:                SetupSucceeded
+Status:                True
+Type:                  Ready-In-Zone-ovn-control-plane Last Transition Time:  2024-06-08T20:29:00Z
+Message:               Setting up OVN DB plumbing was successful
+Reason:                SetupSucceeded
+Status:                True
+Type:                  Ready-In-Zone-ovn-worker
+Last Transition Time:  2024-06-08T20:29:00Z
+Message:               Setting up OVN DB plumbing was successful
+Reason:                SetupSucceeded
+Status:                True
+Type:                  Ready-In-Zone-ovn-worker2
+...
+----
+
+If the plumbing is unsuccessful, you will see an error reported from the respective zone controller on the status.
+
+See the following section for `nbctl` commands to help troubleshoot unsuccessful policies.
```

Comment:
Do you think it would be useful to name the following section? I don't recall seeing something like this in the docs, but I also haven't explicitly looked for it.

Reply:
The commands are actually ovn-nbctl
E.g.
<pre>
oc -n openshift-ovn-kubernetes exec -it ovnkube-node-fqgqn -- ovn-nbctl lr-route-list GR_openshift-qe-027.sriov.openshift-qe.sdn.com
</pre>
@JoeAldinger can use this format where you exec into pod and issue command instead of rsh into pod.

Reply:
@asood-rh I was trying to keep consistency with other procedures in the docs: https://docs.openshift.com/container-platform/4.15/networking/ovn_kubernetes_network_provider/ovn-kubernetes-architecture-assembly.html#nw-ovn-kubernetes-list-database-contents_ovn-kubernetes-architecture:~:text=the%20relevant%20node-,List,-pods%20by%20running